tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -202,11 +202,15 @@ def montecarlo_main_loop(
         )
         rpacket_trackers.append(RPacketTracker())
 
+    # Get the ID of the main thread and the number of threads
     main_thread_id = get_thread_id()
     n_threads = get_num_threads()
 
+    # Create a list of Estimators, one for each thread
     estimator_list = List()
     for i in range(n_threads):  # betting get tid goes from 0 to num threads
```

Comment:
Thanks for clarifying the code here. You should remove this comment since you have corrected it. Do you have a source for nthreads-1?